README.md

```diff
@@ -10,22 +10,15 @@ JerryScript is a lightweight JavaScript engine for resource-constrained devices
 
 Key characteristics of JerryScript:
 * Full ECMAScript 5.1 standard compliance
-* 160K binary size when compiled for ARM Thumb-2
+* [TODO] latest ECMAScript standard compliance
+* [TODO - update] 160K binary size when compiled for ARM Thumb-2
```

Comment:
After these updates, it will good for me.

Reply:
done, please check